frontend/src/app/application-forms/temporary-outfitters/temporary-outfitters.component.ts

```diff
@@ -53,7 +54,8 @@ export class TemporaryOutfittersComponent implements DoCheck, OnInit {
     private router: Router,
     private route: ActivatedRoute,
     public formBuilder: FormBuilder,
-    public renderer: Renderer2
+    public renderer: Renderer2,
```

Comment:
is there another name for this Renderer? Renderer2 does not give me a good indication of what that actually does.

Reply:
@lauraGgit Renderer2 is an angular component: https://angular.io/api/core/Renderer2. We are using it in this controller to add classes to html elements